src/org/infinity/datatype/ColorValue.java

```diff
@@ -131,7 +131,6 @@ public void keyPressed(KeyEvent event)
 
    int selection = Math.min(colorList.getModel().getSize(), Math.max(0, getValue()));
    colorList.setSelectedIndex(selection);
-   select();
```

Comment:
This is not necessary because `edit` for this class called only in `StructViewer.valueChanged`
https://github.com/Argent77/NearInfinity/blob/922dc9e42addddca0fae9d906cb224229baedb38/src/org/infinity/gui/StructViewer.java#L718-L734
and `StructViewer.tableChanged`
https://github.com/Argent77/NearInfinity/blob/922dc9e42addddca0fae9d906cb224229baedb38/src/org/infinity/gui/StructViewer.java#L767-L783

In both places `select` called after `edit`.